### PR TITLE
Adding stock poster Issue #3

### PR DESCRIPTION
**Pull request by DhawalKhapre**
Made changes in the `media.py` .
Adding the stock poster image in the folder

### CODE REVIEW

File: media.py [hunk 1]
```diff
@@ -3,9 +3,15 @@
...
     """ This class provides a way to store movie related information """
     def __init__(self,movie_title,movie_storyline,poster_image,trailer_youtube):
-        self.title = movie_title
-        self.storyline = movie_storyline
-        self.poster_image_url = poster_image
-        self.trailer_youtube_id = trailer_youtube
+        if poster_image == None:
+            self.title = movie_title
+            self.storyline = movie_storyline
+            self.poster_image_url = 'https://thumbs.dreamstime.com/b/movie-film-poster-design-template-background-reel-clapperboard-applicable-to-backdrop-banner-brochure-leaflet-flyer-print-175112902.jpg'
```
Comment (harikesh409) [inline media.py:9]: use some other image that doesn't have watermarks so we will not have any copyright issues.

File: media.py [hunk 2]
```diff
@@ -3,9 +3,15 @@
 class Movie():
     """ This class provides a way to store movie related information """
     def __init__(self,movie_title,movie_storyline,poster_image,trailer_youtube):
-        self.title = movie_title
-        self.storyline = movie_storyline
-        self.poster_image_url = poster_image
-        self.trailer_youtube_id = trailer_youtube
+        if poster_image == None:
+            self.title = movie_title
```
Comment (harikesh409) [inline media.py:7]: the common part in if and else can be moved to outside instead of writing it twice

File: entertainment_center.py [hunk 3]
```diff
@@ -28,9 +28,9 @@ def details(title):
     loaded_json = json.loads(output)
     ID = getTrailerID(title)
     movies.append(media.Movie(loaded_json['Title'],loaded_json['Plot'],loaded_json['Poster'],ID))
-     
 
 for name in names:
     details(name.get('name'))
 
+
```
Comment (harikesh409) [inline entertainment_center.py:35]: can you revert the changes in this file as they are only space changes just don't want to waste a commit for it.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by DhawalKhapre** [comment]
Made a new PR with the required changes
